com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -138,6 +135,48 @@ public delegate void OnAuthLinkExternalAccountCallback(
         //cached log levels for retrieving later
         private static Dictionary<LogCategory, LogLevel> logLevels;
 
+        /// <summary>
+        /// This stores a reference to an existing EOSManager.
+        /// This should only be used to determine if a new copy of EOSManager should become not enabled.
+        /// </summary>
+        private static EOSManager existingManager = null;
+
+        /// <summary>
+        /// Answers the question of whether a new EOSManager should be set up.
+        /// There shouldn't be two active and enabled EOSManagers, this helps prevent accidentally having two.
+        /// There will never be duplicate EOSSingletons, but duplicate EOSManager MonoBehaviours would
+        /// make update ticks run twice when it should run once.
+        /// </summary>
+        private static bool isThereAnExistingRunningManager
```

Comment:
As a C# Property, even if it has a non-public access modifier it should be Pascal Case, meaning it ought to be `IsThereAnExistingRunningManager`.

Reply:
While the above comment is a correct application of the naming convention - I think that this property (moreover the functionality that its existence provides) would be better accomplished by a better implementation of the Singleton pattern - since that is what this property helps to accomplish.